libs/portals/admin/delegation-admin/src/screens/DelegationAdminDetails/DelegationAdmin.tsx

```diff
@@ -10,21 +17,51 @@ import React from 'react'
 import DelegationList from '../../components/DelegationList'
 import { AuthCustomDelegation } from '@island.is/api/schema'
 import { DelegationsEmptyState } from '@island.is/portals/shared-modules/delegations'
+import { useAuth } from '@island.is/auth/react'
+import { AdminPortalScope } from '@island.is/auth/scopes'
 
 const DelegationAdminScreen = () => {
   const { formatMessage } = useLocale()
   const navigate = useNavigate()
   const delegationAdmin = useLoaderData() as DelegationAdminResult
+  const { userInfo } = useAuth()
+
+  const hasAdminAccess = userInfo?.scopes.includes(
+    AdminPortalScope.delegationSystemAdmin,
+  )
 
   return (
     <Stack space="containerGutter">
       <BackButton onClick={() => navigate(DelegationAdminPaths.Root)} />
-      <Box width="full" display="flex" justifyContent="spaceBetween">
-        <IntroHeader
-          title={delegationAdmin.name}
-          intro={delegationAdmin.nationalId}
-        />
-      </Box>
+      <GridRow rowGap={3}>
+        <GridColumn span={['12/12', '8/12']}>
+          <IntroHeader
+            title={delegationAdmin.name}
+            intro={delegationAdmin.nationalId}
+          />
+        </GridColumn>
+        {hasAdminAccess && (
+          <GridColumn span={['12/12', '4/12']}>
+            <Button
+              icon="arrowForward"
+              onClick={() => {
+                const query = new URLSearchParams({
+                  fromNationalId: delegationAdmin.nationalId,
+                })
+
+                navigate(
+                  `${
+                    DelegationAdminPaths.CreateDelegation
```

Comment:
~ We have a function for this called replaceParams

```suggestion
    import { replaceParams } from '@island.is/react-spa/shared'
    
    navigate(
        replaceParams({
            href: DelegationAdminPaths.CreateDelegaion,
            params: { fromNationalId: delegationAdmin.nationalId }
        })
    )
```